apps/mobile/src/screens/welcome-screen/welcome-screen.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { Text, View } from "~/libs/components/components";
+
+const WelcomeScreen: React.FC = () => {
```

Comment:
Maybe Root? Then we will be able to use it in future

Reply:
Not sure that it should be Root. It's just a one of other screens which will be available for authenticated user.